src/HybridSession.php

```diff
@@ -124,7 +114,7 @@ public function gc(int $maxlifetime): int
      *
      * @param string $key Desired session key
      */
-    public static function init($key = null)
+    public static function init(?string $key = null)
```

Comment:
```suggestion
    public static function init(string $key = null)
```
I'm 90% sure this works (if it doesn't, just disregard it) - I'm pretty sure PHP allows null as a default value, but then _doesn't_ allow passing it in explicitly.

If that doesn't work, then set the default to an empty string instead.
We should not be saying `null` is a valid value, and then immediately throw an error when someone passes in `null`.

Reply:
Done